test/github-issues/3375/issue-3375.ts

```diff
@@ -0,0 +1,141 @@
+import "reflect-metadata";
+import {Connection} from "../../../src/connection/Connection";
+import {MigrationExecutor} from "../../../src/migration/MigrationExecutor";
+import {Migration} from "../../../src/migration/Migration";
+import {MongoDriver} from "../../../src/driver/mongodb/MongoDriver";
+import {MongoQueryRunner} from "../../../src/driver/mongodb/MongoQueryRunner";
+import {QueryRunner} from "../../../src/query-runner/QueryRunner";
+import {createTestingConnections, closeTestingConnections} from "../../utils/test-utils";
+import {expect} from "chai";
+
+export class MigrationExecutor2 extends MigrationExecutor {
+    // make this method public just for testing
+    async loadExecutedMigrations(queryRunner: QueryRunner): Promise<Migration[]> {
+        return super.loadExecutedMigrations(queryRunner);
+    }
+}
+
+describe("github issues > #3375 add metadata to migrations table", () => {
+    let connections: Connection[];
+
+    async function createConnections(migrationsPath = __dirname + "/migration/*.js", dropSchema = true) {
+        return createTestingConnections({
+            entities: [__dirname + "/entity/*{.js,.ts}"],
+            migrations: [migrationsPath],
+            enabledDrivers: ["postgres", "mssql", "mysql", "mariadb", "sqlite", "mongodb"],
```

Comment:
If we support migration on all drivers we should remove this option and run tests for all of them.

Reply:
I just removed it. Are CI tests run on Oracle?

Reply:
I just noticed the `mongodb` driver has this set up in both `ormconfig.json` and `ormconfig.travis.json`:

```
    "disabledIfNotEnabledImplicitly": true,
```

Does this mean that tests would not run on mongodb if we remove `enabledDrivers`?